spec/spec_helper.rb

```diff
@@ -239,6 +239,19 @@
     end
   end
 
+  config.around(:each, :no_bullet) do |example|
+    original_enable = Bullet.enable?
+    original_counter_cache_enable = Bullet.counter_cache_enable?
+
+    Bullet.enable = false
+    Bullet.counter_cache_enable = false
+
+    example.run
+  ensure
+    Bullet.enable = original_enable
+    Bullet.counter_cache_enable = original_counter_cache_enable
```

Comment:
See this commit message for discussion of spec flakiness that is introduced if we don't include this line of code: https://github.com/davidrunger/david_runger/pull/4810/commits/4489932347650c325519219214aa7817735ce534